src/app/src/App.js

```diff
@@ -19,16 +19,15 @@ const privateRoutes = (
                 <Route path='/draw' element={<Sidebar />} />
             </Routes>
             <Box flex={1} position='relative'>
-                <Map>
-                    <Routes>
-                        <Route path='/welcome' element={<Welcome />} />
-                        <Route path='/draw' element={<Draw />} />
-                        <Route
-                            path='/'
-                            element={<Navigate to='/welcome' replace />}
-                        />
-                    </Routes>
-                </Map>
```

Comment:
This change undoes https://github.com/azavea/iow-boundary-tool/issues/45. Can we still have Welcome and Draw share the Map?

Reply:
Ooh, thanks. I needed to avoid rendering a Map for the new routes, so I just broke them out in 3e8b84288ca65898b00c784a39cae7c4fd41e166.